lib/ark_elixir/delegate.ex

```diff
@@ -71,22 +71,21 @@ defmodule Ark_Elixir.Delegate do
         "publicKey" => "02c7455bebeadde04728441e0f57f82f972155c088252bf7c1365eb0dc84fbf5de",
         "rate" => 35, "username" => "jarunik", "vote" => "141330551085778"},
         "success" => true}
-    """
-    def get_delegate(username) do
-        Ark_Elixir.Api.get("api/delegates/get", [username: username])
-    end
-
 
-    @doc """
-    Get a single delegate.
-
-    ## Examples
-
-        iex> Ark_Elixir.Delegate.get_delegate("arkValidPublicKey")
-        ...
+        iex> Ark_Elixir.Delegate.get_delegate("02c7455bebeadde04728441e0f57f82f972155c088252bf7c1365eb0dc84fbf5de")
+        %{"delegate" => %{"address" => "Aasu14aTs9ipZdy1FMv7ay1Vqn3jPskA8t",
+        "approval" => 1.18, "missedblocks" => 227, "producedblocks" => 48565,
+        "productivity" => 99.53,
+        "publicKey" => "02c7455bebeadde04728441e0f57f82f972155c088252bf7c1365eb0dc84fbf5de",
+        "rate" => 4, "username" => "jarunik", "vote" => "154019617402053"},
+        "success" => true}
     """
-    def get_delegate(publicKey) do
-        Ark_Elixir.Api.get("api/delegates/get", [publicKey: publicKey])
+    def get_delegate(id) do
+      id_type = case String.length(id) do
```

Comment:
This variable name does not follow your snake case convention. I know that the unofficial Elixir style guide says to use snake case, but I follow what the Elixir source code has with using _ to indicate multiple words. This can be easily changed if needed; I wasn't able to find guidelines on what style to use.

Reply:
With Elixir 1.6 (after it's in stable release), the formatter can be used to get past this issue.